tests/helpers/volunteers.ts

```diff
@@ -138,8 +138,6 @@ export const createVolunteerAndActions = async (): Promise<
   });
 
   const today = new Date();
-  const yesterday = new Date(today);
```

Comment:
Please exclude this file. It is not relevant to the original issue. You have also changed its logic

Reply:
this was included to pass the getVolunteerRanks test, this changed was merged in this [pr](https://github.com/PalisadoesFoundation/talawa-api/pull/2753/files)